scripts/populate-worksection.js

```diff
@@ -0,0 +1,278 @@
+    featured_image.classList.add('featured_image-mobile');
+    featured_image_div.appendChild(featured_image);
+    project_div.appendChild(featured_image_div);
+
+    /* project description (details) */
+    let project_detail_div = document.createElement("div");
+    let name = document.createElement("h2");
+    name.innerHTML=pro.name;
+    name.classList.add('project-title');
+    project_detail_div.appendChild(name);
+    project_div.appendChild(project_detail_div);
+    
+    /* list of languages */
+    let subTitle = makeUL(pro.sub);
+    subTitle.classList.add('icons');
+    project_detail_div.appendChild(subTitle);
+
+    let description = document.createElement('p');
+    description.innerHTML=pro.description;
+    project_detail_div.appendChild(description);
+    
+    let languagesList = makeL(pro.langs);
+    languagesList.classList.add('icons');
+    project_detail_div.appendChild(languagesList);
+
+    let more = document.createElement('button');
+    more.id= pro.id;
+    more.type="button";
+    more.innerHTML="See Project";
+    more.classList.add('see-more-button');
+   
+    project_detail_div.appendChild(more);
+    portfolio.appendChild(project_div);
+}
+
+function addPortfolio(){
+    for(let i= 0; i < projects.length; i++){
+        addWorkSection(projects[i]);
+    }
+}
+
+
+document.addEventListener('DOMContentLoaded', function() {
+    addPortfolio();
+}, false);
+
+// pup up 
+const projectButtons = [];
+const buttons = document.querySelectorAll('button');
+const modal = document.getElementById('myModal');
+
+buttons.forEach((button) => {
+  const [btn] = button.id.split('-');
+  if (btn === 'project') projectButtons.push(button);
+});
+
+const displayModal = (id) => {
+  let li = '';
+  const project = projects.find((p) => p.id === id);
+  project.langs.forEach((project) => {
+    li += `<li class="languages">${project}</li>`;
+  });
+  
+  const render = `
+  <div class="modal-content">
+  <span class="close">&times;</span>
+  <h2 class="modal-card-title">${project.name}</h2>
+  <ul class="icons">
+      <li class="canopy">CANOPY</li>
+      <li class="counter"><span class='dot'> <span>
+      </li>
+      <li class="bg">Back End Dev</li>
+      <li class="counter"><span class='dot'> <span>
+      </li>
+      <li class="bg">2015</li>
+  </ul>
+  <img src=${project.featured_image} alt="featured project image" class="modal-card-img">
+  <div class="modal-project-description">
+      <p class="modal-description">${detailText}</p>
+      <div class="modal-links">
+          <ul class="icons">
+            ${li}
+            <li class="languages">Github </li>
+            <li class="languages">Ruby </li>
+            <li class="languages"> Bootsrap</li>
+          </ul>
+          <hr>
+          <div class="live-links">
+              <button class="btn-enabled" id='live-link' type="button">
+              
+                  see live
+                  <img src="./images/live-icon.png">
```

Comment:
- [ ] In your popup, in the `see live` button, the icon image looks broken and this is because you have used the incorrect path while referencing it. Please check again if you are using the correct name of the image you have in your images folder 😎 👍 
![image](https://user-images.githubusercontent.com/90222110/180577356-f782b085-0f51-42bd-8516-39c1a7ccc70d.png)
💡 **HINT** Can you try `"./images/live-Icon.png"` **instead of "./images/live-icon.png"** 😎 I hope you can see the difference 🏃